Cargo.toml

```
@@ -1,7 +1,15 @@
 [package]
 name = "muse2"
 version = "0.1.0"
```

It's a development version on MUSE in rust, but version 2+ in the big scheme of things, so I would use:
```suggestion
name = "muse"
version = "2.0.0-dev"
```

---

OK yes this seems sensible

---

Neither do I, but whatever its meaning, for sure it's value should not be `2021`!

---

@dalonsoa I think you're right about the version number. I think we should continue to call the project and the resulting binary `muse2` though, because it's basically a totally different piece of software to MUSE v1. I suppose at some point, we may have users with both installed who want both `muse` and `muse2` on their `PATH`.

Bigger picture, I do think it might make sense to give this project a totally new name at some point, given that it's a fresh implementation in a different language and the model is quite different from the one in MUSE v1. That said, I appreciate that it's effectively a decision about branding and MUSE already has some name recognition and a community around it, which you might not want to lose. Just a thought @ahawkes.

Are we happy to just change the version number, but leave the name as `muse2`?

---

Sounds good